src/components/shared/thread/ActivityGraph.js

```diff
@@ -172,7 +189,7 @@ class ThreadActivityGraph extends PureComponent<ActivityGraphProps> {
   }
 
   _onMouseUp = (e: SyntheticMouseEvent<>) => {
-    const activityGraphFills = this._activityGraphFills;
+    const activityGraphFills = this._fillsQuerier;
```

Comment:
Maybe rename this local variable? Or remove it entirely?

Reply:
I fixed the naming, but I need them as local variables to satisfy Flow's null checks. I added a clarifying comment.

```
// Create local variables so that Flow can refine the following to be non-null.
```